code/datums/controllers/process/gang.dm

```diff
@@ -225,13 +258,20 @@
 		var/datum/game_mode/gang/gamemode = ticker.mode
 		var/list/broadcasts = list()
 		for(var/datum/gang/targetGang as anything in gamemode.gangs) //create loot bags for this gang (so they get pinged)
-			if (gang_vandalism_list[targetGang] == null) continue
-			var/area/chosen_area = gang_vandalism_list[targetGang]
-			var/broadcast_string = "<span style='font-size:20px'> Go to the [initial(chosen_area.name)] and wreck it. Cause havoc, cover it in ProPaint tags for a bag!</span>"
-			if (istype(chosen_area, /area/station/janitor))
-				targetGang.vandalism_tracker[chosen_area] = GANG_VANDALISM_REQUIRED_SCORE/2 // usually very small and unlikely to get caught anyway. but funny
+			if (!gang_vandalism_list[targetGang] || gang_vandalism_list[targetGang] == 0) continue
+			var/list/area_names = list()
+			for(var/area/station/chosen_area in gang_vandalism_list[targetGang])
+				area_names += initial(chosen_area.name)
+				targetGang.vandalism_tracker[chosen_area] = 0
+				if (chosen_area.initial_structure_value > 80) //large rooms
+					targetGang.vandalism_tracker_target[chosen_area] = GANG_VANDALISM_BASE_REQUIRED_SCORE
+				else if (chosen_area.initial_structure_value > 50) //medium rooms
+					targetGang.vandalism_tracker_target[chosen_area] = GANG_VANDALISM_BASE_REQUIRED_SCORE/2
+				else
+					targetGang.vandalism_tracker_target[chosen_area] = 2*GANG_VANDALISM_BASE_REQUIRED_SCORE/5
+			if (length(area_names) == 1)
+				broadcasts[targetGang] ="<span style='font-size:20px'> Cover [area_names[1]] in ProPaint tags, and smash it up for a bag!</span>"
 			else
-				targetGang.vandalism_tracker[chosen_area] = GANG_VANDALISM_REQUIRED_SCORE
-			broadcasts[targetGang] = broadcast_string
+				broadcasts[targetGang] = "<span style='font-size:20px'> Cover [english_list(area_names)], in ProPaint tags, and smash them up for loot!</span>"
```

Comment:
You don't need this switch, `english_list` handles that case

Reply:
the latter half of the string is the main reason this is here, it/them

Reply:
ooh I'm blind, oka